addons/netfox.extras/state-machine/network-state.gd

```diff
@@ -0,0 +1,33 @@
+@icon("res://addons/netfox/icons/state-synchronizer.svg")
+extends Node
+class_name NetworkedState
```

Comment:
Once the functionality is solid, I think we could also include a script template for this class, so users can just fill out a skeleton when they create a new state.

Reply:
love this